modules/snooker/src/test/ts/module/ephox/snooker/test/TestGenerator.ts

```diff
@@ -18,10 +19,16 @@ export default (): SimpleGenerators => {
     return r;
   };
 
-  const replace = (name: string) => {
-    const r = 'h(' + name + ')_' + replaceCounter;
-    replaceCounter++;
-    return r;
+  const replace = (elem: SugarElement<Node> | string | number) => {
```

Comment:
I'm surprised this is the only method you had to change? Are the others used by different tests or something?